.github/workflows/cla.yml

```diff
@@ -0,0 +1,31 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+jobs:
+  CLAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        uses: cla-assistant/github-action@v2.1.1-beta
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # the below token should have repo scope and must be manually added by you in the repository's secret
+          PERSONAL_ACCESS_TOKEN : ${{ secrets.CLA_ACCESS_TOKEN }}
+        with:
+          path-to-signatures: 'CLA/signatures/v1.0/cla.json'
+          path-to-cla-document: 'https://github.com/corgibytes/freshli/blob/main/CLA/CLA-v1.0.md'
```

Comment:
We need to check with @mscottford if we we going to have one CLA for all of Freshli or one per repo?  If we have one per repo then the above path will need to point to Freshli-CLI.

Reply:
One per repo seems cleanest.

Reply:
Well, cleanest for us at least. Hopefully, it's not too annoying for others.